src/foam/nanos/script/Script.js

```diff
@@ -166,6 +167,12 @@ foam.CLASS({
       of: 'foam.nanos.auth.User',
       name: 'lastModifiedBy',
       documentation: 'User who last modified script'
+    },
+    {
+      class: 'String',
```

Comment:
Please documentation on why this is necessary and how to set it from Reports for example. Thanks.